src/main/kotlin/net/ccbluex/liquidbounce/utils/inventory/InventoryManager.kt

```diff
@@ -242,10 +242,6 @@ object InventoryManager : EventListener {
 
         if (screen is InventoryScreen || screen is GenericContainerScreen) {
             inventoryOpened()
-
-            if (screen is InventoryScreen) {
-                isInventoryOpenServerSide = true
-            }
```

Comment:
I'm a little bit worried about this.

Reply:
As I thought:
![image](https://github.com/user-attachments/assets/4ff882f9-1fd1-4407-928a-dcd67df734ba)

Inventory Cleaner and other modules will not try to change the items anymore when inventory is opened. But I can easily fix this.

Reply:
Alright, I didn't consider that. I changed it because the server only knows that we're in an inventory if we click a slot.